include/constants/text_formatting.h

```diff
@@ -0,0 +1,26 @@
+#ifndef GUARD_CONSTANTS_TEXT_FORMATTING_H
+#define GUARD_CONSTANTS_TEXT_FORMATTING_H
+
+#define COLOR_1 "#C"
```

Comment:
This file isn't needed because we can just pass it through to the preprocessor with `{}` and `_()` i.e.

`const u8 test_string = _("This is a {COLOR_1 RED} test {END_COLOR_TEXT_1}")`

`src/personality_test.c` does that and is preferred

Reply:
Fixed.